hardening/disa-alignment/oscap/test.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python3
+import os
+import re
+
+from lib import util, results, virt, oscap, versions
+from conf import partitions, remediation
+
+
+virt.Host.setup()
+
+profile = 'xccdf_org.ssgproject.content_profile_stig'
+
+g = virt.Guest('minimal_with_oscap')
+
+if not g.can_be_snapshotted():
+    ks = virt.Kickstart(partitions=partitions.partitions)
+    g.install(kickstart=ks)
+    g.prepare_for_snapshot()
+
+with g.snapshotted():
+    oscap.unselect_rules(util.get_datastream(), 'remediation-ds.xml', remediation.excludes())
+    g.copy_to('remediation-ds.xml')
+
+    for html_report in ['remediation.html', 'remediation2.html']:
```

Comment:
You don't seem to be using these HTML reports - I would either add
```python
g.copy_from(html_report)
```
and add them to `logs=[]`, or (more realistically) just do
```python
for _ in range(2):
    ...
    # remove the --report html_report from oscap CLI
    ...
```